lib/auth.js

```diff
@@ -52,6 +52,25 @@ var auth = {
     } else {
       return cb(null, false);
     }
+  },
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @param  {object} cb
+   * New endpoint required as autentication ($fh.auth) requests were sending 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)